LICENSE-binary

```diff
@@ -515,6 +518,7 @@ org.glassfish.jaxb:jaxb-runtime
 
 Eclipse Public License (EPL) 2.0
 --------------------------------
+io.vertx:vertx-core https://github.com/eclipse-vertx/vert.x
```

Comment:
for dual-licensed 3rd deps, we only need to pick one. for this case, it's under EPL-2.0 and AL 2.0, simply pick the AL 2.0 and no additional LICENSE copy is required.

Reply:
Because these are `dual-licensed 3rd deps`, should we choose `EPL-2.0` or `AL 2.0`?
Is there a `clear standard` for this? Is it still possible to `freely express oneself`?

Reply:
the downstream projects are free to pick one or all licenses they want, as an ASF project, obviously, we prefer to choose AL 2.0

Reply:
> the downstream projects are free to pick one or all licenses they want, as an ASF project, obviously, we prefer to choose AL 2.0

If so, +1

Reply:
+1 for AL 2.0, too.

Reply:
Updated, thanks all!